styles/_footer.scss

```diff
@@ -0,0 +1,13 @@
+.ftr {
+  width: 100%;
+  height: 150px;
+  background: $cyan;
+  font-family: "Montserrat", sans-serif;
+}
+.ftr .icon img {
+  width: 40px;
+  height: 40px;
+  border-radius: 10px;
+  object-fit: cover;
+  cursor: pointer;
+}
```

Comment:
Two things:

Same as above, use descriptive names for classes.
And this isn't _sassy_ enough.

In SCSS we can use nesting. In this case this will be translated to:

```
.ftr {
width: 100%;
  height: 150px;
  background: $cyan;
  font-family: "Montserrat", sans-serif;

.icon {
.img{
  width: 40px;
  height: 40px;
  border-radius: 10px;
  object-fit: cover;
  cursor: pointer;
}
}
}
```